amr-wind/physics/multiphase/MultiPhase.cpp

```diff
@@ -135,7 +205,10 @@ void MultiPhase::levelset2vof()
     const int nlevels = m_sim.repo().num_active_levels();
     const auto& geom = m_sim.mesh().Geom();
 
+    const amrex::Real normal_default = 0.0;
     auto& normal = m_sim.repo().get_field("interface_normal");
+    BCScalar bc_normal(normal);
+    bc_normal(normal_default);
```

Comment:
```suggestion
    normal.set_default_fillpatch_bc(m_sim.time());
```

This should be done once when you declare field, not every time this method is called.